docs/REST/authentication.md

```diff
@@ -6,6 +6,7 @@ All HTTP requests must be authenticated using the **HMAC-SHA256** authentication
 - **Credential** - \<Access Key ID\>
 - **Secret** - base64 decoded Access Key Value. ``base64_decode(<Access Key Value>)``
 
+These values must be obtainted from the Azure App Configuration instance. This can be done using the [Azure Portal](https://portal.azure.com) or the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest). Given a connection string obtained from either one of these sources, one can extract the credential and secret by parsing their respective segments: Endpoint={endpoint};Id={**Credential**};Secret={**Secret**}
```

Comment:
The connection string should be opaque to users. We shouldn't ask users to parse it. This is why both of our portal and CLI list `id` and `secret` separately in addition to the `connection string`. So I will remove the second sentence in the above paragraph. There could be confusion about `credential` and `id`. That's why our portal showed it as `Id (credential)`. We probably can clarify here they are the same thing.

Reply:
Just realized our CLI calls the `secret` as `value`. Is this naming from RP? If so, we can clarify it here too.
```
C:\Windows\System32>az appconfig credential list -n ConfigStoreDemo
[
  {
    "connectionString": "xxxxx",
    "id": "xxxxx",
    "lastModified": "2019-02-25T18:52:32+00:00",
    "name": "Primary",
    "readOnly": false,
    "value": "xxxxx"
  },
 ...
]
```

Reply:
I'll remove the connection string sentence. Although it is an interesting that if we don't document it then people who want to develop their own client can't really leverage it. In other words develop their own AppConfigClient that takes connection string. 

Yes, value comes from the RP. The CLI shows the raw response. I'll clarify.